ChangeLog.md

```diff
@@ -5,8 +5,12 @@
 
   * Add UUIDs to cron job logs for better debugging (Matt West).
 
+  * Add example docker grading images for externally graded questions (James Balamuta).
+  
   * Change v3 questions to disable autocomplete on the question form (Nathan Walters).
 
+  * Change `centos-python` to `grader-python` and place it under `graders/`  (James Balamuta).
```

Comment:
nit: it used to be `centos7-python` (note `7`)

Reply:
```suggestion
  * Change `centos7-python` to `grader-python` and place it under `graders/`  (James Balamuta).
```